README.md

```diff
@@ -4,36 +4,50 @@
 
 Please complete the task detailed in the brief below. 
 
-We are more interested in your approach to solving this problem than if you fully solve it. We also appreciate that your free time is important. Therefore if you feel your code is unfinished please leave some notes in your README.md explaining what you would do next, given more time. We are more than happy to review this.
+We appreciate that your free time is important, and we expect this to take no longer than an evening (2-3 hours) to complete. If you feel your code is unfinished please leave some notes in your README.md explaining what you would do next, given more time. Please take note of the [How We Score](#How-We-Score) section below to ensure that you have demonstrated everything that we are looking for.
 
-When reviewing your project, we'll consider:
+We are looking purely at your technical skill in this exercise. Subsequent interview stages will cover the other aspects that we believe are also important.
 
+# How We Score
+
+When reviewing your project, we'll score it on:
+
+- How much of the task has been completed and how accurately it meets the brief (a following stage in the interview process requires a certain level of functionality)
+- Approach and design patterns used to enable testing. We're looking for code to be written in a way to enable testability and a strong demonstration of how you would test at least one on your created classes/structs in full - the less trivial the better
+- Any bugs that are discovered
```

Comment:
```suggestion
- Any bugs we find in your code
```
Trivial but just reworded as the passive voice could potentially be ambiguous/misleading